lib/ansible/modules/web_infrastructure/jenkins_job.py

```diff
@@ -194,6 +217,16 @@ def __init__(self, module):
         self.EXCL_STATE = "excluded state"
 
     def get_jenkins_connection(self):
+
+        if self.validate_certs == False:
```

Comment:
can you extract this into a separated method? 

So get_jenkins_connection before returning the server object it calls something like:
`self.set_https_context()`

Naming really up to you, but I think it makes sense extracting it into a method.